udata/tests/frontend/test_user_frontend.py

```diff
@@ -50,9 +50,19 @@ def test_render_list_empty(self):
 
     def test_render_profile(self):
         '''It should render the user profile'''
-        user = UserFactory()
+        user = UserFactory(first_name=' John', last_name='Doe   ',
+                           about='* Title 1\n* Title 2',
+                           website='http://www.datagouv.fr/user',
+                           avatar_url='http://www.datagouv.fr/avatar')
         response = self.get(url_for('users.show', user=user))
         self.assert200(response)
+        json_ld = self.get_json_ld(response)
+        self.assertEquals(json_ld['@context'], 'http://schema.org')
```

Comment:
That is right ! It is more complete and, while trying to do it, I even discovered two micro-bugs.

But:
* When the equality is not satisfied, it is harder to debug
* I did not succeed in doing it for the dates, especially 'last_modified'
* It makes sometimes the assertion much heavier

All in all, after having tried to do it, I prefer to keep the existing version.